### PR TITLE
:lipstick:Location  block

### PR DESCRIPTION
**Pull request by Dancunrandy**
# Description
Re-designed location input block
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ 

] This change requires a documentation update

# Screenshot (optional)
Screenshot from 2023-11-05 12-50-39
Screenshot from 2023-11-05 12-54-29

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by LemmyMwaura** [comment]
Update the PR description and add a screenshot

**Comment by Dancunrandy** [comment]
@LemmyMwaura have added the screenshot

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/location-block.jsplumb.ts [hunk 1]
```diff
@@ -22,9 +22,16 @@ export function _LocationBlockDecoratePlumb(block: LocationMessageBlock, comp: C
...
+    maxConnections: -1
+  });
+  jsPlumb.addEndpoint(comp.location.nativeElement, {
+    // Whether the anchor is target (Other Block -> This Block)
+    target: true,
+    cssClass:"block_endpoint",
+    endpoint: "Dot",
+    anchor: [1, 0.9 ,0, 1],
     maxConnections: -1
   });
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/location-block.jsplumb.ts:35]: Remove the extra jsplumb endpoint, return the default option for now.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Kindly Sync your changes and resolve conflicts

**Comment by Dancunrandy** [comment]
@LemmyMwaura resolved the conflicts and  enabled default option, take a look.

**Pull request closed without merging**